### ISSUE TITLE
What about top-k loss of softmax cross entropy?

### ISSUE THREAD

**Issue by zhaoguangxiang** [garcinc/noised-topk#2]
Are there existing losses? Or what is the barrier to achieving it?
